frontend/src/utility/dumb.ts

```diff
@@ -11,4 +12,33 @@ export async function getFirstNode(nodeManager: NodeManager): Promise<string> {
     }
 
     return myNode
-}
\ No newline at end of file
+}
+
+export enum PaymentType {
+    "invoice",
+    "keysend",
+    "onchain",
+    "unknown"
+}
+
+export function detectPaymentType(p: string): PaymentType {
+    if (p.startsWith("lnt")) {
+        return PaymentType.invoice
+    } else if (p.startsWith("03") || p.startsWith("02")) {
+        return PaymentType.keysend
+    } else if (p.startsWith("tb1")) {
+        return PaymentType.onchain
```

Comment:
the invoice and on-chain ones won't work for mainnet. Also on-chain wont work for older addresses. might be a good idea to bring in a library here, I imagine we'll eventually have, on-chain address, invoice, node pubkey, lnurl, and lightning address, so library could potentially simplify this